### DISCUSSION TITLE
可不可以用xml格式定义beans？

### DISCUSSION THREAD

**Discussion by SinForLove** [focus-creative-games/luban#110] [General]
![image](https://github.com/focus-creative-games/luban/assets/78003316/87349d9c-6ab8-455e-9c4f-a480f0cb8696)
我在luban.conf 文件里加了一行{"fileName":"Datas/b2.xml", "type":"bean"},， 写了个b2.xml文件定义bean ,但是转表时提示
2024-01-3116:52:58.6188|===>"can't find schema 1oader for type :bean extName : xmn1"

![image](https://github.com/focus-creative-games/luban/assets/78003316/6e0a418d-7aad-4cf2-ae6b-61902fb792b9)
所以说是需要自己写一个schema loader吗？

**Comment by DustinMaple** [comment]
把你的xml放到Defines目录就行了。扫描xml只支持陪目录名。

**Reply by SinForLove** [reply]
谢谢！  但是我把b2.xml 放到defines 文件夹里后，转表又报了
定义文件:E:/unityFrogram/LubanDemo/Lubanlable/DataTables/Defines/b2.xm module:DemoType2不支持 tag :var"
这个错误 ，这是我照着文档写的
![image](https://github.com/focus-creative-games/luban/assets/78003316/e5eb5cb6-98c1-473b-9fbb-54cb11ea02fd)

**Reply by DustinMaple** [reply]
xml最外层标签一定要是module.
`
<module name="">
    <bean name="DemoType2">
    </bean>
</module>
`
